gapis/service/service.proto

```diff
@@ -420,6 +421,21 @@ message MemoryRange {
   uint64 size = 2;
 }
 
+// UsageHints hints to the server the intended usage of the result of a request.
+// This can be used to improve performance and responsiveness of the RPCs.
+message UsageHints {
```

Comment:
I supposed there is precedent for "usage hints" to be bitfield.  Which this effectively is.

I bet Ian talked you into going with something that does not have clear ordering...